source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -227,6 +262,21 @@ content: |
            {
                { "mongocryptdSpawnPath", $@"{MongoBinariesPath}\mongocryptd.exe" },
            };
+
+     .. tab::
```

Comment:
Question: is there an option to bypass spawn if it's already running? If so, we should include it here similar to in some of the other driver tabs.

Reply:
There is an option, yep. I must have missed including it in that section.